docs/tutorials/get-started-with-ngo.md

```diff
@@ -0,0 +1,619 @@
+
+The client can request a new position, instructing the server to change that instance's position `NetworkVariable`. After the server updates the position `NetworkVariable`, the client applies that `NetworkVariable` position inside its `Update()` method.
+
+## Add a `NetworkTransform`
+
+This section guides you through adding a `NetworkTransform` component that moves the player.
+
+Add a `NetworkTransform` component to the Player prefab:
+
+1. Select the **Player** prefab in Assets > Prefabs.
+2. In the **Inspector** tab (with the Player prefab selected), select **Add Component**.
+3. Select **Netcode** > **NetworkTransform**.
+
+Create a script named `NetworkTransformTest.cs`.
+
+1. In the **Project** tab, go to **Assets** > **Scripts**.
+2. Right-click, then select **Create** > **C# Script**.
+3. Name it `NetworkTransformTest`.
+
+Add the `NetworkTransformTest` script to the Player prefab:
+
+1. Select the **Player** prefab in **Assets** > **Prefabs**.
+2. In the **Inspector** tab (with the Player prefab selected), select **Add Component**.
+3. Select **Scripts** > **Network Transform Test**.
+
+Edit the `NetworkTransformTest.cs` script:
+
+1. In the **Project** tab, select **Assets** > **Scripts** > **`NetworkTransformTest`**.
+2. In the **Inspector** tab (with the script selected), select **Open**. This opens this script in the default local text editor.
+3. Edit the `NetworkTransformTest.cs` script to match the following:
+
+```csharp
+using System;
+using Unity.Netcode;
+using UnityEngine;
+
+public class NetworkTransformTest : NetworkBehaviour
+{
+    void Update()
+    {
+        if (IsServer)
+        {
+            float theta = Time.frameCount / 10.0f;
+            transform.position = new Vector3((float) Math.Cos(theta), 0.0f, (float) Math.Sin(theta));
+        }
+    }
+}
+```
+
+4. Save the scene by selecting **File** > **Save**.
+
+### Test the `NetworkTransform`
+
+This section guides you through testing the `NetworkTransform` you added in the earlier section.
+
+1. Select **File** > **Build And Run**.
+2. Stop the player.
+3. Launch the client and server together in a terminal as shown in [Test the command line helper](command-line-helper).
+
+After the client and server spawn, you should see the player capsule moving in a circle on both the client and the server.
+
+## Test Hello World
+
+To see if everything works as expected, test starting a host in the Unity Editor. A host plays the role of a server and a client at the same time.
+
+You can test your Hello World project through the Unity Editor or a command line helper. If you choose the latter, see [Create a command line helper](command-line-helper). Otherwise, see the following instructions to test through the Unity Editor.
+
+1. Select **Play** from the top of the Unity Editor to start the scene.
+
+2. Select **NetworkManager** from the **Hierarchy** list.
+
+3. With NetworkManager selected, select **Start Host** from the **Inspector** tab.
```

Comment:
At this point of the tutorial we already have the in-game buttons, so we don't really need to use the ones in NetworkManager's inspector. This could still be useful at the point where we only have the player prefab and the plane setup.

Reply:
I'll add a note that they can use either. Or do you think I should favor the in-game buttons?

Reply:
since one aim of the HelloWorldManager script was to replace the inspector buttons, I think we should favor the in-game ones